src/rpcrawtransaction.cpp

```diff
@@ -808,9 +808,9 @@ UniValue sendrawtransaction(const UniValue& params, bool fHelp)
         throw JSONRPCError(RPC_DESERIALIZATION_ERROR, "TX decode failed");
     uint256 hashTx = tx.GetHash();
 
-    bool fOverrideFees = false;
-    if (params.size() > 1)
-        fOverrideFees = params[1].get_bool();
+    CFeeRate rawMaxFeeRate = CFeeRate(::minRelayTxFee.GetFeePerK() * 10000);
```

Comment:
What about DEFAULT_TRANSACTION_MAXFEE from #7084 ?
In fact, why not unify both PRs and make both things at once?